### DISCUSSION TITLE
Issue with vim.lsp.buf.rename() ?

### DISCUSSION THREAD

**Discussion by cj** [NvChad/NvChad#1180] [General]
Hi,

Is anyone else having issues using `:lua vim.lsp.buf.rename()` with the latest NvChad and neovim 7?

![image](https://user-images.githubusercontent.com/1819/172301285-9be3625d-0490-43d2-9dc5-e2aef2665e31.png)

and when I run `:lua vim.lsp.buf.rename()` in a ts file on anything I get the following error:

```
E5108: Error executing lua [string ":lua"]:1: attempt to call field 'rename' (a table value)
stack traceback:
        [string ":lua"]:1: in main chunk
```

All other lsp commands work.

**Comment by siduck** [comment]
@cj fixed,  howeever you can use our renamer fucntion too

![image](https://user-images.githubusercontent.com/59060246/172304623-8c23dbd1-82b0-4611-989d-a7cc98770e01.png)


```lua
require('ui.renamer').open()
```

this is mapped to leader ra
